src/client/apps/shared/styled.ts

```diff
@@ -18,7 +18,8 @@ export const StyledRenderingAreaWrapper = styled("div")(({ theme }) => ({
 	display: "flex",
 	alignItems: "center",
 	justifyContent: "center",
-	padding: theme.spacing(1),
+	//
+	// padding: theme.spacing(1),
```

Comment:
Is this still needed?

Reply:
hm, not really, I was testing. it removes the paddingarouind the image-zoom, please see if the behavior is better without padding, then this can be removed.